### PR TITLE
User description for ndenys650 added

### PR DESCRIPTION
**Pull request by ndenys650**
Description:
In this Pull Request we have done the following additions/edits for the code:

Note: adding my user description to participants.
#1 referenced!

### CODE REVIEW

File: PARTICIPANTS/ndenys650.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Name: Nate Denys
```
Comment (MJ10) [inline PARTICIPANTS/ndenys650.txt]: Name of the directory should be `Participants`.

**Comment by salman-bhai** [comment]
Are there any updates or should we close this PR as `spam`?

**Comment by salman-bhai** [comment]
spam

**Comment by ndenys650** [comment]
my bad guys. I didn't see that this had comments till today. keep up the hard work.

**Pull request closed without merging**